### ISSUE TITLE
Support import on aws_backup_vault resource

### ISSUE THREAD

**Issue by martcollins** [terraform-providers/terraform-provider-aws#8821]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The aws_backup_vault resource doesn't yet allow import.  This would be useful for users with existing backup vaults.  Issue found using version 2.12 of the provider.

```
$ terraform import aws_backup_vault.test-vault test-vault
aws_backup_vault.test-vault: Importing from ID "test-vault"...

Error: aws_backup_vault.test-vault (import id: test-vault): import aws_backup_vault.test-vault (id: test-vault): resource aws_backup_vault doesn't support import
```

### New or Affected Resource(s)

* aws_backup_vault

### Potential Terraform Configuration

```hcl
resource "aws_backup_vault" "test-vault" {
  name = "TestVault"
}
```

**Comment by programmer04** [comment]
My PR #9041 adds it, but there is a bug in AWS API for non-existent `backup_vault`, returns the misleading error code (`403` instead of `404`). Bug's been reported to AWS [here](https://forums.aws.amazon.com/thread.jspa?threadID=301836&tstart=0) and should be fixed, but it is independent of merging this PR.

**Comment by bflad** [comment]
Support for importing `aws_backup_vault` resources has been merged and will be released in version 2.16.0 of the Terraform AWS Provider, likely tomorrow. 👍 Thanks, @programmer04!

**Comment by martcollins** [comment]
Thanks @programmer04, awesome!

**Comment by bflad** [comment]
This has been released in [version 2.16.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2160-june-20-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
